templates/ts-fastify-mongodb-backend/package.json

```diff
@@ -14,15 +14,16 @@
   "license": "Apache 2.0",
   "dependencies": {
     "@graphback/runtime-mongo": "1.0.1",
-    "apollo-server-express": "2.17.0",
     "cors": "2.8.5",
     "dotenv": "8.2.0",
-    "express": "4.17.1",
+    "fastify": "^3.5.0",
```

Comment:
Remove ^

Reply:
Didn't get you? :sweat_smile: Remove what? I've already removed express no?